### PR TITLE
Styling update

### PR DESCRIPTION
**Pull request by peduarte**
Some copy updates to the Styling overview page.

### CODE REVIEW

File: pages/primitives/docs/overview/styling.mdx [hunk 1]
```diff
@@ -4,132 +4,168 @@ description: Radix Primitives are unstyled and unopinionated by default.
...
+    </Item>
+  </Accordion>
+);
 ```
+
+### Conclusion
+
+Radix Primitives were designed to encapsulate accessibility concerns and other complex functionalities, while ensuring consumers have full control over the look and feel.
+
+To ensure compatibility with Global CSS solutions, components include a `data-radix-*` attribute. Stateful components include a `data-state` attribute.
```
Comment (chaance) [inline pages/primitives/docs/overview/styling.mdx:169]: `Global` can be lowercased

File: pages/primitives/docs/overview/styling.mdx [hunk 2]
```diff
@@ -4,132 +4,168 @@ description: Radix Primitives are unstyled and unopinionated by default.
...
+
+Every component that renders a HTML element will contain these attributes, so feel free to inspect them and look around.
```
Comment (chaance) [inline pages/primitives/docs/overview/styling.mdx:171]: an* HTML element

**Review by chaance — CHANGES_REQUESTED** [review]
Two minor things, looks good otherwise

File: pages/primitives/docs/overview/styling.mdx (hunk 1 above)
Comment (peduarte) [reply pages/primitives/docs/overview/styling.mdx:169]: yes thats very true! I hate randomly uppercased wors haha

File: pages/primitives/docs/overview/styling.mdx (hunk 2 above)
Comment (peduarte) [reply pages/primitives/docs/overview/styling.mdx:171]: legend

**Review by chaance — APPROVED** [review]

**Pull request merged**